pkg/controller/networkpolicy/validate.go

```diff
@@ -167,11 +217,16 @@ func (v *NetworkPolicyValidator) validateTier(curTier, oldTier *secv1alpha1.Tier
 			reason = fmt.Sprintf("cannot delete reserved tier %s", oldTier.Name)
 			return reason, false
 		}
-		// Tier with existing CNPs cannot be deleted
+		// Tier with existing CNPs/ANPs cannot be deleted
 		cnps, err := v.networkPolicyController.cnpInformer.Informer().GetIndexer().ByIndex(TierIndex, oldTier.Name)
 		if err != nil || len(cnps) > 0 {
-			allowed = false
-			reason = fmt.Sprintf("tier %s is referenced by %d NetworkPolic(y)ies", oldTier.Name, len(cnps))
+			reason = fmt.Sprintf("tier %s is referenced by %d ClusterNetworkPolicies", oldTier.Name, len(cnps))
+			return reason, false
+		}
+		anps, err := v.networkPolicyController.anpInformer.Informer().GetIndexer().ByIndex(TierIndex, oldTier.Name)
+		if err != nil || len(anps) > 0 {
+			reason = fmt.Sprintf("tier %s is referenced by %d NetworkPolicies", oldTier.Name, len(anps))
```

Comment:
you use "Antrea NetworkPolicies" in other places

and when possible I would actually recommend using "Antrea ClusterNetworkPolicies" in log messages for consistency. What do you think?

Reply:
done